libretro/libretro.cpp

```diff
@@ -321,10 +321,14 @@ void retro_init(void) {
 	g_Config.bFrameSkipUnthrottle = false;
 	g_Config.bMemStickInserted = PSP_MEMORYSTICK_STATE_INSERTED;
```

Comment:
This is a boolean, should just be true.

-[Unknown]

Reply:
Will change it next upstream PR